test/framework/src/main/java/org/elasticsearch/cluster/project/TestProjectResolvers.java

```diff
@@ -60,7 +61,7 @@ public boolean supportsMultipleProjects() {
      * with the executeOnProject method. This is mostly useful in places where we just need a placeholder to satisfy
      * the constructor signature.
```

Comment:
Nit: I think we should remove the last sentence. The "placeholder usage" is `singleProjectOnly`. `mustExecuteFirst` has a pretty niche use case.

Reply:
Yeah it's a leftover that should be removed.